src/components/navigation-header/components/product-page-content/index.tsx

```diff
@@ -111,11 +111,16 @@ const ProductPageHeaderContent = () => {
 			color: 'highlight' as const,
 		},
 	}
-	const allMainMenuItems = [
-		{ items: [homeMenuItem] },
-		{ items: betaProductItems, label: 'Products' },
-		{ items: comingSoonProductItems, label: 'Coming Soon' },
-	]
+
+	// Construct item groups for the dropdown, avoid adding empty groups
+	const allMainMenuItems = []
+	allMainMenuItems.push({ items: [homeMenuItem] })
```

Comment:
Since this `push` will always be invoked, can this be updated to instantiate the array with this first item?

```typescript
const allMainMenuItems: NavigationHeaderItemGroup[] = [
  { items: [homeMenuItem] },
]
```

Reply:
Absolutely! ✅ updated